src/utils/test-utils.ts

```diff
@@ -0,0 +1,17 @@
+import { SwaggerV2, SwaggerV3 } from '../types';
+
+export const swaggerV3Mock = (schemas: any): SwaggerV3 => {
+    return {
+        openapi: '3.0.0',
+        components: {
+            schemas,
+        },
+    };
+};
+
+export const swaggerV2Mock = (definitions: any): SwaggerV2 => {
```

Comment:
Our mock function usually start with `a` or `an`
Probably rename those to `aSwaggerV3Mock` and `aSwaggerV2Mock` ?

Reply:
@ardeois Sure, will do